tiltak/src/test/kotlin/no/nav/amt/tiltak/tiltak/controllers/GjennomforingControllerIntegrationTest.kt

```diff
@@ -1,216 +1,217 @@
-
-		val deltakere = controller.hentDeltakere(gjennomforing.id)
-		val deltaker1 = deltakere.get(0)
-
-		deltakere.size shouldBe 1
-		deltaker1.fodselsnummer shouldBe bruker1.fodselsnummer
-		deltaker1.fornavn shouldBe bruker1.fornavn
-		deltaker1.etternavn shouldBe bruker1.etternavn
-
-		deltaker1.oppstartdato shouldBe startDato
-		deltaker1.sluttdato shouldBe sluttDato
-		deltaker1.registrertDato.truncatedTo(ChronoUnit.MINUTES) shouldBe regDato.truncatedTo(ChronoUnit.MINUTES)
-
-	}
-
-	@Test
-	fun `hentDeltakere - Flere deltakere finnes`() {
-		val arrangorId = insertArrangor()
-		val tiltak = tiltakRepository.insert((1000..9999).random().toString(), tiltakKode, tiltakKode)
-
-		val gjennomforing = insertGjennomforing(tiltak.id, arrangorId)
-		val bruker1 = BrukerInsertDbo("12128673847", "Person", "En", "To", "123", epost, null, null)
-		val bruker2 = BrukerInsertDbo("12128674909", "Person", "En", "To", "123", epost, null, null)
-
-		insertDeltaker(
-			gjennomforing.id,
-			bruker1,
-			LocalDate.now().minusDays(3),
-			LocalDate.now().plusDays(1),
-			LocalDateTime.now().minusDays(10)
-		)
-		insertDeltaker(
-			gjennomforing.id,
-			bruker2,
-			LocalDate.now().minusDays(3),
-			LocalDate.now().plusDays(1),
-			LocalDateTime.now().minusDays(10)
-		)
-
-		val deltakere = controller.hentDeltakere(gjennomforing.id)
-		val deltaker1 = deltakere.get(0)
-		val deltaker2 = deltakere.get(1)
-
-		deltakere.size shouldBe 2
-		deltaker1.fodselsnummer shouldBe bruker1.fodselsnummer
-		deltaker2.fodselsnummer shouldBe bruker2.fodselsnummer
-	}
-
-	private fun insertDeltaker(
-		gjennomforingId: UUID,
-		bruker: BrukerInsertDbo,
-		startDato: LocalDate,
-		sluttDato: LocalDate,
-		regDato: LocalDateTime
-	): DeltakerDbo {
-		val bruker = brukerRepository.insert(bruker)
-		return deltakerRepository.insert(
-			brukerId = bruker.id,
-			gjennomforingId = gjennomforingId,
-			oppstartDato = startDato,
-			sluttDato = sluttDato,
-			status = Deltaker.Status.GJENNOMFORES,
-			dagerPerUke = 5,
-			prosentStilling = 10f,
-			registrertDato = regDato
-		)
-	}
-
-	private fun insertGjennomforing(tiltakId: UUID, arrangorId: UUID): GjennomforingDbo {
-		var arenaId = (1000..9999).random()
-		return gjennomforingRepository.insert(
-			arenaId = arenaId,
-			tiltakId = tiltakId,
-			arrangorId = arrangorId,
-			navn = "Kaffekurs",
-			status = Gjennomforing.Status.GJENNOMFORES,
-			oppstartDato = null,
-			sluttDato = null,
-			registrertDato = null,
-			fremmoteDato = null
-		)
-	}
-
-	private fun insertArrangor(): UUID {
-		val id = UUID.randomUUID()
-		val orgnr = (1000..9999).random()
-		val insert = """
-			INSERT INTO arrangor(id, overordnet_enhet_organisasjonsnummer, overordnet_enhet_navn, organisasjonsnummer,navn)
-			VALUES ('$id', '12345678', 'Orgnavn1', '$orgnr', 'Virksomhetsnavn1')
-		""".trimIndent()
-		namedJdbcTemplate.jdbcTemplate.update(insert)
-
-		return id
-	}
-
-}
+package no.nav.amt.tiltak.tiltak.controllers
```

Comment:
Hvorfor ser det ut som du har endra på linjer som er like?

Reply:
Vanskelig å se hva som har blitt gjort her

Reply:
Tror egentlig ikke det er gjort noen funksjonelle endringer. Kan hende at filen byttet line ending eller no sånt

Reply:
Ja, jeg hadde samme problem når jeg tenker meg om. Ved commit får man nå spm om å sette default for line ending og hvis man trykker ok så setter den globale innstillinger for line endings i github config. Kan være lurt å fjerne denne igjen

Reply:
Line endingen er riktig nå, sjekket på main og det var CRLF, mens vi bruker LF i resten av filene